editor.planx.uk/src/routes/views/flowEditor.tsx

```diff
@@ -79,11 +84,18 @@ export const getFlowEditorData = async (
  */
 export const flowEditorView = async (req: NaviRequest) => {
   const [flow] = req.params.flow.split(",");
-  const { id, flowAnalyticsLink, isFlowPublished, isTemplate, templatedFrom } =
-    await getFlowEditorData(flow, req.params.team);
+  const {
+    id,
+    status,
```

Comment:
This was the cause of the bug, we're assigning `status` to the store, but only `flowStatus` is the expected / known property.

I'll take a look to see if we can make our Zustand stores stricter so that assigning a variable like this will get caught by TypeScript.